guides/cloud/aws_certification_advice.md

```diff
@@ -0,0 +1,105 @@
+
+#### Recommended Online Training
+1. [Cantrill.io](https://learn.cantrill.io/p/aws-certified-sysops-administrator-associate)
+2. [ACloudGuru](https://learn.acloud.guru/course/aws-certified-sysops-administrator-associate)
+
+### AWS Certified DevOps Engineer - Professional 
+
+The [AWS Certified DevOps Engineer - Professional](https://aws.amazon.com/certification/certified-devops-engineer-professional/) is the last step in the DevOps Pro certification pathway, covering the following domains. SDLC Automation, Configuration Management & Infrastructure as Code, Monitoring & Logging, Policies & Standards Automation, Incident & Event Response, High Availability, Fault Tolerance & Disaster Recovery.
+
+#### Recommended Online Training
+1. [Stephane Maarek - Udemy](https://www.udemy.com/course/aws-certified-devops-engineer-professional-hands-on/)
+2. [ACloudGuru](https://acloudguru.com/course/aws-certified-devops-engineer-professional)
+
+
+## AWS Solutions Architect Pathway
+
+After following the basic training and achieving the AWS Certified Cloud Practitioner Certification, Engineers working towards achieving the AWS Certified Solutions Architect - Professional should follow this pathway.
+
+This pathway only consists of the Associate and Pro level Solutions Architect exams, but I would advise looking at other the Associate level courses before undertaking the AWS Solutions Architect Pro, is it is very difficult exam!
+
+### AWS Certified Solutions Architect - Associate
+
+The [AWS Certified Solutions Architect - Associate](https://aws.amazon.com/certification/certified-solutions-architect-associate/) is the Associate level Solutions Architect Certification and covers the following domains. Design Resilient Architectures, Design High-Performing Architectures, Design Secure Application & Architectures and Design Cost-Optimisted Architectures.
+
+#### Recommended Online Training
+1. [Cantrill.io](https://learn.cantrill.io/p/aws-certified-solutions-architect-associate-saa-c02)
+2. [ACloudGuru](https://learn.acloud.guru/course/aws-certified-solutions-architect-associate)
+
+### AWS Certified Solutions Architect - Professional
+
+The [AWS Certified Solutions Architect - Professional](https://aws.amazon.com/certification/certified-solutions-architect-professional) is the professional level Solutions Architect Certification and probably the most difficult AWS Exam. It covers the following domains. Design for Organisational Complexity, Design for New Solutions, Migration Planning, Cost Control and Continuous Improvement for Existing Solutions.
+
+#### Recommended Online Training
+1. [Cantrill.io](https://learn.cantrill.io/p/aws-certified-solutions-architect-professional)
+2. [ACloudGuru](https://learn.acloud.guru/course/aws-certified-solutions-architect-professional/dashboard)
+
+
+## AWS Speciality Certifications
```

Comment:
We may want to hold back on specifying these until we know what we want to prioritise based on propositions and aligning to competencies that support Jonathans efforts to grow our partner offerings.

Reply:
Sure, this was just a first pass/WIP. I was going to pad out the speciality areas in a second pass :-) you know how agile I am :-)